code/controllers/subsystem/ticker.dm

```diff
@@ -469,15 +469,15 @@ SUBSYSTEM_DEF(ticker)
 			if(Player.stat != DEAD && !isbrain(Player))
 				num_survivors++
 				if(station_evacuated) //If the shuttle has already left the station
-					var/area/shuttle_area
+					var/list/area/shuttle_areas
 					if(SSshuttle && SSshuttle.emergency)
-						shuttle_area = SSshuttle.emergency.areaInstance
+						shuttle_areas = SSshuttle.emergency.shuttle_areas
 					if(!Player.onCentcom() && !Player.onSyndieBase())
 						to_chat(Player, "<font color='blue'><b>You managed to survive, but were marooned on [station_name()]...</b></FONT>")
 					else
 						num_escapees++
 						to_chat(Player, "<font color='green'><b>You managed to survive the events on [station_name()] as [Player.real_name].</b></FONT>")
-						if(get_area(Player) == shuttle_area)
+						if(get_area(Player) in shuttle_areas)
```

Comment:
shuttle_area used to be the instanced area that was the shuttle. shuttle_areas is a list of instanced areas that makes up the shuttle.